src/meta/src/hummock/manager/versioning.rs

```diff
@@ -116,38 +110,21 @@ impl Versioning {
         min_pinned_version_id
     }
 
-    pub fn extend_objects_to_delete_from_deltas(
-        &mut self,
-        delta_range: impl RangeBounds<HummockVersionId>,
-        metric: &MetaMetrics,
-    ) {
-        self.extend_objects_to_delete_from_deltas_impl(delta_range);
-        trigger_stale_ssts_stat(metric, self.objects_to_delete.len());
-    }
-
-    /// Extends `objects_to_delete` according to given deltas.
-    /// Possibly extends `deltas_to_delete`.
-    fn extend_objects_to_delete_from_deltas_impl(
+    /// Marks all objects <= `min_pinned_version_id` and in `range` for deletion.
+    pub(super) fn mark_objects_for_deletion(
         &mut self,
-        delta_range: impl RangeBounds<HummockVersionId>,
+        range: (Bound<HummockVersionId>, Bound<HummockVersionId>),
```

Comment:
It seems the right bound is unneccesary since it always equals to `self.checkpoint.id`